classification/algorithms/SequenceClassifier.py

```diff
@@ -451,172 +366,32 @@ def compute(self, recordNum, patternNZ, classification, learn, infer):
         else:
           self._actualValues[bucketIdx] = actValue
 
-      # Train each pattern that we have in our history that aligns with the
-      # steps we have in self.steps
-      for nSteps in self.steps:
-
-        # Do we have the pattern that should be assigned to this classification
-        # in our pattern history? If not, skip it
-        found = False
-        for (iteration, learnPatternNZ) in self._patternNZHistory:
-          if iteration == self._learnIteration - nSteps:
-            found = True;
-            break
-        if not found:
-          continue
-
-        # Store classification info for each active bit from the pattern
-        # that we got nSteps time steps ago.
-        for bit in learnPatternNZ:
+      # Train pattern.
+      # Store classification info for each active bit.
+      for bit in patternNZ:
 
-          # Get the history structure for this bit and step #
-          key = (bit, nSteps)
-          history = self._activeBitHistory.get(key, None)
-          if history is None:
-            history = self._activeBitHistory[key] = BitHistory(self,
-                        bitNum=bit, nSteps=nSteps)
+        # Get the history structure for this bit 
+        history = self._activeBitHistory.get(bit, None)
+        if history is None:
+          history = self._activeBitHistory[bit] = BitHistory(self,
+                      bitNum=bit)
```

Comment:
i don't this it is. the indent seems correct to me. the outer loop `for nStep in self.steps` has been removed on purpose here